test/starknet-verify/hardhat.config.ts

```diff
@@ -0,0 +1,7 @@
+import "../dist/index.js";
+
+module.exports = {
+    paths: {
+        starknetArtifacts: "my-starknet-artifacts"
```

Comment:
why not just use the default starknetArtifacts path? like in the `plain` test case